app/(tabs)/(home)/index.tsx

```diff
@@ -118,21 +150,79 @@ export default function HomeScreen() {
     );
   };
 
-  function StopButton() {
+  const StopButton = () => {
     return (
       <BigRoundButton
         icon={
           <FontAwesomeIcon icon="stop" size={buttonIconSize} color="white" />
         }
         onPress={() => {
           setButtonState(ButtonStates.NotCycling);
-          stopAutomaticTracking();
+          void stopAutomaticTracking();
         }}
       />
     );
-  }
+  };
+
+  const showRoute = async () => {
+    if (geoJSON) {
+      // Find the outermost coordinates
+      let minLat = Infinity,
+        maxLat = -Infinity,
+        minLng = Infinity,
+        maxLng = -Infinity;
+      geoJSON.features.forEach((feature) => {
+        if (feature.geometry.type === "LineString") {
+          (feature.geometry as GeoJSON.LineString).coordinates.forEach(
+            ([lng, lat]) => {
+              if (lat < minLat) minLat = lat;
+              if (lat > maxLat) maxLat = lat;
+              if (lng < minLng) minLng = lng;
+              if (lng > maxLng) maxLng = lng;
+            },
+          );
+        } else if (feature.geometry.type === "Point") {
```

Comment:
Ist es nötig, die Punkte auch noch zu überprüfen? Das sind doch nur Start und Ziel, und die sind doch eh in den Linien mit drin? Oder verstehe ich da was falsch?

Reply:
Manchmal liegen die Punkte abseits der Route. Hatte das vorher auch nicht drin. Aber das schadet nicht auszurechnen